Docs: Add callout for variables behavior with classic conditions (#67253)
grafana/grafana: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md

The first condition is {{ $values.B0 }}, and the second condition is {{ $values.B1 }}
```

> With Classic Conditions, labels from the query is not available in `$labels` variable because it generates single alert instance. Instead, you can retrieve the labels from `$values` variable.
> 

Reviewer: ```suggestion
With classic conditions, labels from the query are not available in `$labels` variable, because single alert instance are generated. Instead, you can retrieve the labels from the `$values` variable.

```
